src/packages/mikroorm/src/database.ts

```diff
@@ -1,4 +1,3 @@
-import 'reflect-metadata';
```

Comment:
I think we might actually need this in the MikroORM package only.

Reply:
I think we are ok as there is a ref at the top of `src/packages/mikroorm/src/index.ts`